client/my-sites/stats/stats-purchase/stats-purchase-notice.jsx

```diff
@@ -21,6 +22,10 @@ const handleUpgradeClick = ( event, upgradeUrl, isOdysseyStats ) => {
 		? recordTracksEvent( 'jetpack_odyssey_stats_purchase_summary_screen_upgrade_clicked' )
 		: recordTracksEvent( 'calypso_stats_purchase_summary_screen_upgrade_clicked' );
 
+	trackStatsAnalyticsEvent( 'stats_upgrade_clicked', {
+		type: 'summary_screen',
```

Comment:
Every other places are all using hyphens not underscores. Maybe we could consolidate on that.